docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -80,6 +80,36 @@ At a minimum, ``hive.metastore.uri`` must be configured:
     - Maximum number of partitions handled per writer.
     - 100
 
+.. _iceberg-authorization:
+
+Authorization checks
+^^^^^^^^^^^^^^^^^^^^
+
+You can enable authorization checks for the connector by setting
+the ``iceberg.security`` property in the catalog properties file. This
+property must be one of the following values:
+
+.. list-table:: Iceberg security values
+  :widths: 30, 60
+  :header-rows: 1
+
+  * - Property value
+    - Description
+  * - ``ALLOW_ALL``
+    - No authorization checks are enforced.
+  * - ``SYSTEM``
+    - The connector relies on system-level access control.
+  * - ``READ_ONLY``
+    - Operations that read data or metadata, such as :doc:`/sql/select` are
+      permitted. No operations that write data or metadata, such as
+      :doc:`/sql/create-table`, :doc:`/sql/insert`, or :doc:`/sql/delete` are
+      allowed.
+  * - ``FILE``
+    - Authorization checks are enforced using a configuration file whose path
+      is specified in the ``security.config-file`` catalog configuration
+      property. See :ref:`hive-file-based-authorization` for information on
```

Comment:
It would be nice to move this out of `hive` as now it becomes more generic.

Reply:
Agreed, but there's a lot of restructuring work that needs to be done to the Hive docs. There's already too many different docs covering these topics.

Reply:
Followed up with https://github.com/trinodb/trino/pull/10800